### DISCUSSION TITLE
Welcome to vue3-baidu-map-gl Discussions!

### DISCUSSION THREAD

**Discussion by yue1123** [yue1123/vue3-baidu-map-gl#17] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kingsaj1211** [comment]
请问一下该组件库，支持自定义瓦片服务功能吗？如果支持改怎么样实现？
开发环境：vue3、ts、vite、node版本v18.16.0

**Reply by yue1123** [reply]
组件库没有做相关的封装，需要自己通过 BMapGL api 来实现这个功能
参考： 
https://lbsyun.baidu.com/jsdemo.htm#custom
https://yue1123.github.io/vue3-baidu-map-gl/zh-CN/guide/com-events

**Comment by lly134** [comment]
我想问一下大家，如何使用vue3 +百度离线地图
